helix-view/src/input.rs

```diff
@@ -387,6 +387,21 @@ impl std::str::FromStr for KeyEvent {
                     .then_some(KeyCode::F(function))
                     .ok_or_else(|| anyhow!("Invalid function key '{}'", function))?
             }
+            _ if s.ends_with('-')
+                && tokens.last().map(|s| str::is_empty(s)).unwrap_or_default() =>
+            {
+                if tokens.len() > 1 {
+                    return Err(anyhow!(
+                        "Key '-' cannot be used with modifiers, use '{}' instead",
+                        keys::MINUS
+                    ));
```

Comment:
I like the explicit error for this case recommending using `keys::MINUS`. We can expand on it a little by recommending the full key sequence:

```rust
let mut suggestion = s[..s.len() - 1].to_string();
suggestion.push_str(keys::MINUS);
```

and suggesting that in the error message